gitws/git.py

```diff
@@ -391,6 +391,12 @@ def get_revision(self) -> Optional[str]:
         """
         return self.get_branch() or self.get_tag() or self.get_sha()
 
+    def get_upstream_branch(self) -> Optional[str]:
+        """Get Actual Upstream Branch."""
```

Comment:
Is the "Actual" here a "false friend" and should be "current"?

Reply:
you are right, own you a beer ;-)